week-4/implement/is-prime.test.js

```diff
@@ -1,3 +1,47 @@
 // Given a positive integer num,
 // When the isPrime function is called with num as input,
 // Then it should return a boolean representing whether the num is prime
+
+function isPrime(num) {
+  if (isNaN(num) || num <= 0 || Number.isInteger(num) !== true) {
```

Comment:
Just small refactoring is possible here: `!Number.isInteger(num)` instead of `Number.isInteger(num) !== true`